docs/configuration.md

```diff
@@ -78,6 +78,8 @@ telemetries: [
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | stackTraceLength | Number | `200` | The number of characters to record from a JavaScript error's stack trace (if available). |
+| errorsToExclude | RegExp[] | `[]` | A list of error messages that will be used to identify errors to ignore. Matched errors will not be recorded. |
```

Comment:
Initially thought of accepting both string and regex, but seems like the pattern we have is accepting only regex input (whether intentional or unintentional). Decided to follow what we have.

Reply:
One thing that this PR doesn't contain is live update of list of errors to ignore. I personally don't think it would be a useful add-on to ignore errors, so I didn't add it to this scope.

Reply:
When my initial prototyping had both string and regex, I had to convert the string inputs to `RegExp` objects to use the matching methods (if this is what you're referring to!)